use-timescale/data-tiering/enabling-data-tiering.md

```diff
@@ -38,9 +35,181 @@ console.
 
 </Procedure>
 
-After tiered storage is enabled you must either [manually tier data][manual-tier-chunk] or [setup a tiering policy][creating-data-tiering-policy] 
-to begin tiering data from your hypertables.
+## Automate tiering with policies
+
+A tiering policy automatically moves data to the object storage tier. Any chunks that only contain data
+older than the `move_after` threshold are moved. This works similarly to a
+[data retention policy][data-retention], but chunks are moved rather than deleted. You can add tiering policies to hypertables, including continuous aggregates.
+
+A tiering policy schedules a job that runs periodically to migrate eligible chunks. The migration is asynchronous. The chunks are considered tiered once they appear in the `timescaledb_osm.tiered_chunks` view. Tiering does not influence your ability to query the chunks.
+
+### Add a tiering policy
+
+To add a tiering policy, call `add_tiering_policy`:
+
+```sql
```

Comment:
remove

Reply:
remove what?

Reply:
The bits that are moved to the updates i suggested previously.